code/addons/jest/src/components/Result.tsx

```diff
@@ -86,5 +86,3 @@ export function Result(props: ResultProps) {
     </Fragment>
   );
 }
-
-export default Result;
```

Comment:
This isn't a component that users might import directly, right?

Reply:
can't, it's bundled